src/Compilers/CSharp/Test/Semantic/SourceGeneration/StateTableTests.cs

```diff
@@ -96,7 +96,7 @@ public void Node_Builder_Can_Add_Entries_From_Previous_Table()
             Assert.True(didRemoveEntries);
         }
 
-        private IEnumerable<int> YieldItems(OneOrMany<int> items)
+        private static IEnumerable<int> YieldItems(OneOrMany<int> items)
```

Comment:
nit: do we need this at all?

Reply:
alternative recommendation?

Reply:
Places that use this can use `GetEnumerator` instead of `YieldItems`, no?

Reply:
unfortunately no, that returns an Enumerator/IEnumerator.  The test helpers test IEnumerables.  I could make OneOrMany IEnumerable, but it seemed excessive (and potentially worrying for boxing reasons) just for test purposes.